homeassistant/components/foscam/__init__.py

```diff
@@ -86,24 +82,3 @@ def update_unique_id(entry):
     LOGGER.debug("Migration to version %s successful", entry.version)
 
     return True
-
-
-async def async_migrate_entities(hass: HomeAssistant, entry: FoscamConfigEntry) -> None:
-    """Migrate old entry."""
-
-    @callback
-    def _update_unique_id(
-        entity_entry: RegistryEntry,
-    ) -> dict[str, str] | None:
-        """Update unique ID of entity entry."""
-        if (
-            entity_entry.domain == Platform.SWITCH
-            and entity_entry.unique_id == "sleep_switch"
-        ):
-            entity_new_unique_id = f"{entity_entry.config_entry_id}_sleep_switch"
-            return {"new_unique_id": entity_new_unique_id}
-
-        return None
-
-    # Migrate entities
-    await async_migrate_entries(hass, entry.entry_id, _update_unique_id)
```

Comment:
We should not remove this migration as this has been added only recently and this might still be needed